packages/framework/src/Model/Statistics/ValueByDateTimeDataPointFormatter.php

```diff
@@ -19,15 +19,15 @@ public function __construct(protected readonly DateTimeFormatterExtension $dateT
 
     /**
      * @param \Shopsys\FrameworkBundle\Model\Statistics\ValueByDateTimeDataPoint[] $valueByDateTimeDataPoints
-     * @param \DateTime $startDateTime
-     * @param \DateTime $endDateTime
+     * @param \DateTimeImmutable $startDateTime
+     * @param \DateTimeImmutable $endDateTime
      * @param \DateInterval $interval
      * @return array
      */
     public function normalizeDataPointsByDateTimeIntervals(
         array $valueByDateTimeDataPoints,
-        DateTime $startDateTime,
-        DateTime $endDateTime,
+        DateTimeImmutable $startDateTime,
+        DateTimeImmutable $endDateTime,
```

Comment:
We have DateTimeImmutable here and DateTimeInterface in the ValueByDateTimeDataPoint class